Dockerfile.e2e

```diff
@@ -1,7 +1,8 @@
-FROM golang:1.7
-ADD GLOCKFILE src/github.com/dcoker/biscuit/
+FROM golang:1.16
```

Comment:
The README.md currently refers to 1.6. Please update the version in README.md to match the Dockerfile.

Reply:
Updated! Thanks!